spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/SnapshotTableSparkAction.java

```diff
@@ -174,7 +174,7 @@ protected Map<String, String> destTableProps() {
 
     // make sure we mark this table as a snapshot table
     properties.put(TableProperties.GC_ENABLED, "false");
```

Comment:
As `GC_ENABLED` is set to false (line 176) for this table created by snapshot action, expire snapshots and remove_orphan files will anyways not clean up the files. So, I am wondering why do we need one property that does the same thing?

Reply:
I think you said is right, I need to rethink this. Thanks for sharing your thought.